packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -713,7 +713,11 @@ class RenderIntrinsicHeight extends RenderProxyBox {
 
 }
 
-int _getAlphaFromOpacity(double opacity) => (opacity * 255).round();
+/// Returns an alpha value representative of the provided [opacity] value.
```

Comment:
Is this a generally useful function? If it is generally useful, it seems like something that really doesn't have that much to do with proxy_box, and probably belongs in `painting.dart` in `dart:ui` alongside the `Color` class, instead of here.

FYI, there's an identical version of this in `flow.dart`, so it seems that it's been needed before.